Cargo.toml

```diff
@@ -36,9 +36,7 @@ bytes = "1.2.1"
 either = "1.8.0"
 embedded-hal = { version = "0.2.7", features = ["unproven"], optional = true }
 embedded-svc = { version = "0.26.4", optional = true }
-esp-idf-hal = { version = "0.42.5", optional = true }
-esp-idf-svc = { version = "0.47.3",  optional = true }
-esp-idf-sys = { version = "0.33.7", features = ["binstart"],  optional = true }
+esp-idf-svc = { version = "= 0.47.3",  optional = true }
```

Comment:
Also also, note that `esp-idf-svc` is now explicitly pinned so we don't get unexpected upgrades.